app/models/apartment.rb

```diff
@@ -1,3 +1,4 @@
 class Apartment < ApplicationRecord
     belongs_to :user
+    validates :street, :unit, :city, :state, :square_footage, :bathrooms, :bedrooms, :pets, :image,:price, presence: true 
```

Comment:
To make sure that each apartment is associated with a user, the presence of the foreign key needs to be required also